### PR TITLE
Bug/intygfv 14940 - Patient address can be editable

### PR DESCRIPTION
**Pull request by SopraErik**

### CODE REVIEW

File: packages/webcert/src/feature/certificate/PatientAddress/PatientAddress.tsx [hunk 1]
```diff
@@ -20,62 +45,107 @@ const CityInput = styled(TextInput)`
 `
 
 const PatientAddress: React.FC = () => {
+  const isShowValidationError = useSelector(getShowValidationErrors)
+  const validationErrors = useSelector(getCareUnitValidationErrors(), _.isEqual)
```
Comment (ChrilleSigerhed) [inline packages/webcert/src/feature/certificate/PatientAddress/PatientAddress.tsx:49]: Should this be patientValidationErrors?

Comment (SopraErik) [reply packages/webcert/src/feature/certificate/PatientAddress/PatientAddress.tsx]: of course. very observant! Fixed now.

**Review by ChrilleSigerhed — CHANGES_REQUESTED** [review]
I checked out the branch and did some checks to see how the implemented logic works. Here is a listing of my findings.

1. Unable to type in the component in a user friendly way, seems related to dispatch.
2. The field postaddress should be expandable: https://inera.atlassian.net/wiki/spaces/IT/pages/9805812/I+-+WC+Inputkomponenter+standard#6.1-Postadress
3. The asterix seems missplaced, this is same for "vardenhet" on the bottom of the page, would a fix for this make sense?

File: packages/common/src/utils/validationUtils.ts [hunk 2]
```diff
@@ -40,6 +40,11 @@ export const CARE_UNIT_CITY_FIELD = 'grunddata.skapadAv.vardenhet.postort'
 export const CARE_UNIT_PHONE_NUMBER_FIELD = 'grunddata.skapadAv.vardenhet.telefonnummer'
 export const CARE_UNIT_ADDRESS_CATEGORY_TITLE_ID = 'vardenhetensadress'
 export const CARE_UNIT_ADDRESS_CATEGORY_TITLE = 'Vårdenhetens adress'
+export const PATIENT_STREET_FIELD = 'grunddata.patient.postadress'
```
[the remark this answers is not in the document]
Comment (SopraErik) [reply packages/common/src/utils/validationUtils.ts:43]: I can't seem to provoke any vissible error messages for the care unit either, maybe there aren't any?, but I've adjusted the components and client validation and inputs for patient address to be similar to care unit

**Review by ChrilleSigerhed — APPROVED** [review]
Nice, looks great!

**Pull request merged**